platform/accessibility/guidance/staging-review-processes.md

```diff
@@ -0,0 +1,51 @@
+# Accessibility Staging Review Processes
+Accessibility staging reviews are part of the wider [VSP staging review](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=andreahewitt-odd%2C+meganhkelley&labels=product+support%2C+content-ia-team&template=staging-review.md&title=Staging+Review+%5BFeature-Name%5D) process. The accessibility review is a bit unique because it involves a number of automated and manual tests, and may include a simple code review.
+
+All staging reviews will test the [happy path](https://en.wikipedia.org/wiki/Happy_path) through the application or content page(s). Staging reviews **may not catch all accessibility issues**, but will ensure basic functionality is accessible. Accessibility specialists will conduct a more thorough audit when your application is relatively code-stable in production.
+
+## Before You Request a Review
+Staging accessibility reviews will be turned around in two days, provided the following items are done ahead of time:
+
+* 1-2 sprints before the review needs to be completed, provide a "heads up" in Slack to the accessibility specialists that the staging review request will be coming soon
```

Comment:
VFS teams don't necessarily run on the same sprint schedule we do; can we replace "1-2 sprints" with a concrete length of time?

Reply:
Updated to 2-4 weeks.